### PR TITLE
Explicitly set owner of authorized_keys file for SSH ops user

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR explicitly sets the owner of `~cyhy_ops/.ssh/authorized_keys` to `cyhy_ops:cyhy_ops` when the file is created.

It also pins Terraform to 0.12.x and the AWS provider to 2.x until we can fully test the newer versions.

## 💭 Motivation and Context

A recent change to Ansible made `~cyhy_ops/.ssh/authorized_keys` be owned by `root:root` and unreadable by the `cyhy_ops` user when it was created.  This was preventing successful SSH connections from `cyhy_ops` users with authorized keys.

## 🧪 Testing
I verified the fix by changing the owner of `~cyhy_ops/.ssh/authorized_keys` to `root:root`, then doing a targeted apply of `module.cyhy_bastion_ansible_provisioner.null_resource.provisioner` and verified that `~cyhy_ops/.ssh/authorized_keys` ownership was correctly set to `cyhy_ops:cyhy_ops`. 

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: ansible/roles/cyhy_ops/tasks/main.yml [hunk 1]
```diff
@@ -25,4 +25,6 @@
     path: /home/cyhy_ops/.ssh/authorized_keys
     line: "{{ lookup('aws_ssm', '/ssh/public_keys/{{ item }}') }}"
     create: yes
+    owner: cyhy_ops
+    group: cyhy_ops
```
Comment (mcdonnnj) [inline ansible/roles/cyhy_ops/tasks/main.yml:29]: Should we explicitly set the `mode` as well for completeness?

**Review by mcdonnnj — APPROVED** [review]
Thanks for taking care of this 💪

File: ansible/roles/cyhy_ops/tasks/main.yml (hunk 1 above)
Comment (dav3r) [reply ansible/roles/cyhy_ops/tasks/main.yml:29]: It couldn't hurt- done in https://github.com/cisagov/cyhy_amis/pull/275/commits/47f9887f656a4cb1da9078c40dbfd3fe452dd071.  Thanks for the suggestion.

**Review by jsf9k — APPROVED** [review]
Great save! :football:

**Comment by dav3r** [comment]
Resolves #274

**Pull request merged**